Sources/Core/IOData/directory_listing_entry.cpp

```diff
@@ -39,7 +39,7 @@ class DirectoryListingEntry_Impl
 {
 //! Construction:
 public:
-	DirectoryListingEntry_Impl()
+	DirectoryListingEntry_Impl():	directory(false), hidden(false), writable(false), readable(false)
```

Comment:
It would better to change this to a C++11'ish style where it simply says "directory = false;" down where the member is declared.